apps/beeswax/src/beeswax/server/hive_server2_lib.py

```diff
@@ -226,14 +226,23 @@ def details(self):
     if self._details is None:
       props = dict([(stat['col_name'], stat['data_type']) for stat in self.properties if stat['col_name'] != 'Table Parameters:'])
       serde = props.get('SerDe Library:', '')
+      format = ''
```

Comment:
Line no used/needed

Reply:
Removed. And I renamed `format` to `details_format` since `format` will shadow Python build-in variable.